test/model.jl

```diff
@@ -382,6 +382,11 @@ function test_model()
         model = Model(with_optimizer(MOIU.MockOptimizer, mock))
         JuMP.set_time_limit(model, 12.0)
         @test JuMP.time_limit(model) == 12.0
+        JuMP.set_time_limit(model, nothing)
+        @test JuMP.time_limit(model) === nothing
+        JuMP.set_time_limit(model, 12.0)
```

Comment:
Test that getting time limit here give 12

Reply:
Isn't that done by line 384? Or shell I do it after setting the time limit to nothing as well? I though `nothing` is the standard so this wouldn't test anything new or am I wrong?

Reply:
Yes, indeed, I was just reviewing the diff and didn't see the above lines